administration/live-migration.adoc

```diff
@@ -32,6 +32,8 @@ data:
   feature-gates: "DataVolumes,LiveMigration"
 ....
 
+Additionally, Virtual machines must have a PersistentVolumeClaim (PVC) with a shared ReadWriteMany (RWX) access mode to be live migrated.
```

Comment:
@iranzo Thanks for this clarification. However, I think we should further clarify it as having a PVC is not a requirement for live migration - it can have any of the supported disks. Only if the VM uses a PVC it should be RWX.

Reply:
ok, let me rephrase

Reply:
Thanks :)